testsuite/python-imagebufalgo/src/test_imagebufalgo.py

```diff
@@ -264,6 +264,11 @@ def test_iba (func, *args, **kwargs) :
     b = test_iba (ImageBufAlgo.invert, a)
     write (b, "invert.tif", oiio.UINT8)
 
+    # normalize
+    a = ImageBuf (OIIO_TESTSUITE_ROOT+"/common/vectorschart_raw.tif")
+    b = test_iba (ImageBufAlgo.normalize, a, 0.5, 0.5, 0.5)
+    write (b, "normalize.tif", oiio.UINT16)
```

Comment:
You'll also need to add normalize.tif to the list of outputs checked in testsuite/python-imagebufalgo/run.py, and the correct output in the ref/ subdirectory.

Reply:
One last thing: vectorschart_raw.tif doesn't need to be in "common", I would just put it directly in testsuite/python-imagebufalgo because it's only used for this test.

Reply:
Oh, unless you intend to use vectorschart_raw.tif for the oiiotool test as well; in that case, maybe common is fine.

Reply:
Yes, i think it can be used in oiiotools tests too